src/qmapshack/gis/trk/CGisItemTrk.cpp

```diff
@@ -506,6 +506,17 @@ QString CGisItemTrk::getInfo(quint32 feature) const
             IUnit::self().meter2elevation(totalDescent, val2, unit2);
 
             str += tr(", %1%2%3, %4%5%6").arg(QChar(0x2197)).arg(val1).arg(unit1).arg(QChar(0x2198)).arg(val2).arg(unit2);
+
+            if(actCnt == 1 && acts.contains(CTrackData::trkpt_t::eAct20Foot))
+            {
+                str += "<br />";
+                //ominal walking time according to DIN 33466
+                qreal elevationTime = totalAscent/300 * 60 + totalDescent/500 * 60;
+                qreal distanceTime = totalDistance/4000 * 60;
+                qreal nominalTime = qMax(elevationTime, distanceTime) + 0.5 * qMin(elevationTime, distanceTime);
+                IUnit::self().seconds2time(nominalTime, val1, unit1);
+                str += tr("Nom. Time: %1%2").arg(val1).arg(unit1);
```

Comment:
I doubt that any user will understand what `Nom. Time` is about.

Reply:
Yes, I doubt that too.
However, I wanted to shorten it, so it wouldn't take up that much space in the info box.
Any other suggestion?

Reply:
`est. walk. time` An move it below the total time and moving time.